spec/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require './spec/controllers/api/v1/authenticated_user'
+
+describe Api::V1::BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'When creating a valid book suggestion with authorized spanish user' do
+      let(:user) { create(:user, :lang_es) }
+      let!(:book_suggestion_attr) { attributes_for(:book_suggestion) }
+
+      before do
+        request.headers.merge! user.create_new_auth_token
+        post :create, params: { book_suggestion: book_suggestion_attr }
+      end
+
+      it 'creates a new book_suggestion' do
+        expect do
+          request_response = JSON.parse(response.body)
+          expect(request_response['user_id']).to eq user.id
+        end
+      end
+
```

Comment:
U should test that the bookSuggestion user_id is the same as the user_id of the test

Reply:
This response is returning the bookSuggestion that i create in the "it" block, why is this wrong?

Reply:
¿Why does your description not match the test then? It should be `creates a new book_suggestion with a valid user_id` for example 😄